ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/configs/DynamoDBConfigs.java

```diff
@@ -0,0 +1,42 @@
+package com.netflix.ndbench.plugin.dynamodb.configs;
+
+import com.netflix.archaius.api.annotations.Configuration;
+import com.netflix.archaius.api.annotations.DefaultValue;
+
+/**
+ * Configurations for DynamoDB benchmarks
+ *
+ * @author ipapapa
+ */
+@Configuration(prefix = "ndbench.config.dynamodb")
```

Comment:
Just to follow the standard conventions, can you change the prefix to use
`NdBenchConstants.PROP_NAMESPACE +`

Reply:
I saw this too late and I merged it. I will update it in an upcoming PR.